source/manual/get-started.html.md

```diff
@@ -135,11 +156,38 @@ On GOV.UK we use the following command-line tools for AWS and SSH access:
     gds config email <FIRSTNAME>.<LASTNAME>@digital.cabinet-office.gov.uk
     ```
 
-1. Run `gds config yubikey false` if you use your phone as an Multi-Factor Authentication (MFA) device.
+1. Set up AWS credentials:
+
+    1. [Create an AWS access key][create-aws-access-key] via the [console][gds-users-aws-signin].
+    1. Run a GDS CLI command to prompt for credentials. For example `gds aws govuk-integration-readonly -l`.
+    1. Enter your Access Key ID and Secret Access Key
+    1. Enter your AWS MFA token
+    1. When prompted, save credentials to your Mac's keychain as `aws-vault` and set a password for the keychain. Save that password somewhere safe, for example in a password manager.
+
+    Here is an example of the output you'll see:
+
+    ```shell
+    $ gds aws govuk-integration-readonly -l
+    Welcome to the GDS CLI! We will now store your AWS credentials in the keychain using aws-vault.
+    Enter Access Key ID: <YOUR-ACCESS-KEY-ID>
+    Enter Secret Access Key: <YOUR-SECRET-ACCESS-KEY>
+    Added credentials to profile "gds-users" in vault
+    Successfully initialised gds-cli
+    Enter token for arn:aws:iam::123456789012:mfa/firstname.lastname@digital.cabinet-office.gov.uk: 123456
+    ```
+
+If you have a GDS-issued Yubikey, you can run `gds config yubikey true` in the GDS CLI to set GDS CLI to automatically pull the MFA code from your Yubikey.
+
+You have completed the get started process. You can now use `gds aws` to run generic [aws CLI](https://aws.amazon.com/cli/) commands by prefixing them with `gds aws <role>`. For example:
+
+```shell
+$ gds aws govuk-integration-readonly aws s3 ls
```

Comment:
nit: omitting the `$` prompt makes the copy button in GitHub more useful (though devdocs doesn't have a copy button yet so it's moot for now, but hey)
```suggestion
gds aws govuk-integration-readonly aws s3 ls
```

Reply:
Good shout - having the prompts in docs is pet peeve of mine.